src/containers/App/test/App.test.js

```diff
@@ -114,15 +114,6 @@ describe('App container', () => {
     })
   })
 
-  it('renders account page', () => {
```

Comment:
Why was this test removed?

Reply:
its a duplicate - already being tested in the accouts page tests

Reply:
Let's leave it in for now - this is testing routing.

Reply:
the test also fails - but ill put it back in